server/src-lib/Hasura/Server/Auth/JWT.hs

```diff
@@ -73,23 +75,94 @@ data JWTClaimsFormat
 $(J.deriveJSON J.defaultOptions { J.sumEncoding = J.ObjectWithSingleField
                                 , J.constructorTagModifier = J.snakeCase . drop 3 } ''JWTClaimsFormat)
 
-data JWTConfigClaims
+defaultClaimsFormat :: JWTClaimsFormat
+defaultClaimsFormat = JCFJson
+
+allowedRolesClaim :: SessionVariable
+allowedRolesClaim = mkSessionVariable "x-hasura-allowed-roles"
+
+defaultRoleClaim :: SessionVariable
+defaultRoleClaim = mkSessionVariable "x-hasura-default-role"
+
```

Comment:
Probably, can keep the `defaultRoleClaim` and the `allowedRolesClaim` to be `Text` to avoid converting it back to text (needed in lots of places). But, the reason they are a `SessionVariable` is to make sure that the text is in lower-case. So, the current approach is safe, but it comes at some repetition of code.